src/text_format.cc

```diff
@@ -30,6 +30,7 @@ bool ParseTextMessage(const std::string& data, protobuf::Message* output) {
   TextFormat::Parser parser;
   parser.SetRecursionLimit(100);
   parser.AllowPartialMessage(true);
+  parser.AllowUnknownField(true);
   if (!parser.ParseFromString(data, output)) {
     output->Clear();
```

Comment:
could you please also add trivial test?

Reply:
Where should I add it? In `mutator_test.cc`?

Reply:
Yes

Reply:
There is serialization test

Reply:
Done